src/index.html

```diff
@@ -7,6 +7,511 @@
+      <div class="logo">
+        <a href="#page-top">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a href="#" class="nav__link is-active">APPLE</a>
+          </li>
+          <li class="nav__li">
+            <a href="#" class="nav__link">SAMSUNG</a>
+          </li>
+          <li class="nav__li">
+            <a href="#" class="nav__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav__li">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="#" class="nav__link">ГАДЖЕТЫ</a>
+          </li>
+          <li class="nav__li">
+            <a href="#" class="nav__link">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="nav__li">
+            <a href="#" class="nav__link">ФОТО</a>
+          </li>
+          <li class="nav__li">
+            <a href="#" class="nav__link">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__row">
+        <section class="section" data-qa="card-hover">
+          <div class="section__container-image">
+            <img
+              src="images/imac.jpeg"
+              alt="computer"
+              class="section__image"
+              height="134px"
+              width="160px"
+            >
+          </div>
+          <div class="section__name-code-about section__name-code-about--margin">
+            <div class="section__name section__name--margin">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </div>
+
+            <div class="section__name-code-about-margin-code">
+              <div class="section__code">
+                Код товара: 195434
+              </div>
+            </div>
+          </div>
+
+          <div class="rate section__rate">
+            <div class="rate__stars rate__stars--4">
+              <div class="rate__star rate__star--margin"></div>
+              <div class="rate__star rate__star--margin"></div>
+              <div class="rate__star rate__star--margin"></div>
+              <div class="rate__star rate__star--margin"></div>
+              <div class="rate__star rate__star--margin"></div>
+            </div>
+
+            <div class="rate__feedback">
+              Отзывов: 5
+            </div>
+          </div>
+
+          <div class="section__about-price">
+            <div class="section__price-word">
+              Цена:
+            </div>
+
+            <div class="section__price">
+              69 999 грн
+            </div>
+          </div>
+
+          <div class="section__button-container">
+            <a
+              href="#buy"
+              class="section__button"
+              data-qa="hover"
```

Comment:
Set correct attribute: 
```suggestion
              data-qa="card-hover"
```